docs/change-existing-WDL-for-Azure.md

```diff
@@ -0,0 +1,59 @@
+# How to modify an existing WDL file to run on Cromwell on Azure
+
+For any pipeline, you can create a [WDL](https://software.broadinstitute.org/wdl/) file that calls your tools in docker containers. Please note that Cromwell on Azure only supports tasks with docker containers defined for security reasons.<br/>
+
+In order to run a WDL file, you must modify/create a workflow with the following runtime attributes for the tasks that are compliant with the [TES or Task Execution Schemas](https://cromwell.readthedocs.io/en/develop/backends/TES/):
+
+```
+runtime {
+    cpu: 1
+    memory: 2 GB
+    disk: 10 GB
+    docker:
+    maxRetries: 0
+}
+```
+Ensure that the attributes `memory` and `disk` (note: use the singular form for `disk` NOT `disks`) have units. Supported units from Cromwell:
+
+> KB - "KB", "K", "KiB", "Ki"<br/>
+> MB - "MB", "M", "MiB", "Mi"<br/>
+> GB - "GB", "G", "GiB", "Gi"<br/>
+> TB - "TB", "T", "TiB", "Ti"<br/>
+
+`preemptible` and `zones` attributes are currently not being passed through Broad's Cromwell to the TES backend, and hence are not supported.<br/>
```

Comment:
preemptible is now passed through; Kris' commit was merged :-) https://github.com/broadinstitute/cromwell/commit/37e4a116b6140bd35b520bfa43afa63d660da5e1